core/errors.go

```diff
@@ -29,6 +30,8 @@ func isTemporary(info httperror.Info, err error) bool {
 		return true
 	case "CH001": // request timed out
 		return true
+	case "CH012": // conflict
+		return true
```

Comment:
Currently the only conditionals we use are IfNotExists when storing a config. That seems like a "Your operation conflicted" error, so I updated this to make this error permanent. When we add an "an internal operation conflicted and you should retry" error case, we can add the temporary error.

This has me thinking though, the IfNotExists conditional makes /configure not idempotent. Maybe we should be using conditional that allows the operation if the key doesn't exist or its value is equal to x. Can we compare two protobufs using proto.Equal without knowing the schema? If not, gogo/protobuf has a [stable_marshaler option](https://github.com/gogo/protobuf/pull/156).